Informed/Lesson Two/README.md

```diff
@@ -0,0 +1,381 @@
+<!-- Generics / Data Structures -->
+
+Welcome back! [Last time](../Lesson One), we took a look at Swift's syntax and Xcode playgrounds. We touched _very_ briefly on the idea of types. 
+
+Swift is a _strongly typed language_ and the type system is very robust. Combined with type inference, you can write expressive code without being repetitive. Many languages might define an integer variable as `int i = 0`, but in Swift, it's just `var i = 0`, and Swift figures out that `0` is an integer. 
+
+In this lesson, we're going to take a whirlwind tour through Swift's different types:
+
+- Structs
+- Classes
+- Enums
+- Tuples
+
+Finally, we'll take a deeper look at generics. Let's get started!
+
+----------------
+
+**Structs** in Swift are containers for other data, similar to structs in C. This is a valid Swift struct.
```

Comment:
Fair enough – it was more to help anyone who _has_ seen them in C. I'll expand on the definition at the beginning of the sentence.